index.html

```diff
@@ -0,0 +1,260 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>McLaren</title>
+  <link href="style.css" rel="stylesheet" type="text/css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+  
+</head>
+
+  <body>
+  
+    <header>
+
+      <div class="headerLog">
+        <div class="headerLogFacebook"><i class="fa-brands fa-facebook-f"></i></div>
+        <div class="headerLogTwitter"><i class="fa-brands fa-twitter"></i></div>
+        <div class="headerLogLanguage"><a href="">English</a></div>
+        <div class="headerLogLanguage"><a href="">My Page</a></div>
+        <div class="headerLogout"><a href="">Logout</a></div>
+      </div>
+
+      <nav>
+        <div class="navLogo"><img src="images/mclarenlogo..png"></div>
+        <div class="nav"><a href="about.html">About</a></div>
+        <div class="nav"><a>Program</a></div>
+        <div class="nav"><a>Join</a></div>
+        <div class="nav"><a>Sponsor</a></div>
+        <div class="nav"><a>News</a></div>
+        <div class="navCampaign"><a>Campaign</a></div>
+      </nav>
+    </header>
+
+    <main>
+     
+      <section id="landing">
+        
+        <div class="landing-container">
+
+          <div class="menu-mobile" ><i class="fa-solid fa-bars" id="trigger"></i></div>
+
+          <div id="menumobile">
+            <div class="menuClose"><i class="fa-solid fa-square-xmark" id="menuClose"></i></div>
+            <ul>
+              <li><a href="#works" id="menuOption1">Portfolio</a></li>
+              <li><a href="#about" id="menuOption2">About</a></li>
```

Comment:
- [ ] Kindly change the href value of this <a> tag to `about.html` so that the page is migrated to about.html when user clicks on this option.